bucket/virt-viewer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "10.0-1.0",
+    "description": "Open source tool to connect to virtual machines to display VNC or SPICE",
+    "homepage": "https://virt-manager.org/download/",
+    "license": "GPL v3+",
+    "architecture": {
+        "64bit": {
+            "url": "https://virt-manager.org/download/sources/virt-viewer/virt-viewer-x64-10.0-1.0.msi",
+            "hash": "f50c13602060184b3e2db5f5822ff1860ac2eec39660578774008892e3516166"
+        },
+        "32bit": {
+            "url": "https://virt-manager.org/download/sources/virt-viewer/virt-viewer-x86-10.0-1.0.msi",
+            "hash": "ada7d13136f8ec2cf574071aa0bc1c1b8cf14ee62614b0728cdce5a46b080934"
+        }
+    },
+    "extract_dir": "VirtViewer v10.0-256",
+    "bin": [
+        "bin\\virt-viewer.exe",
+        "bin\\remote-viewer.exe"
+    ],
```

Comment:
what are available command line arguments?

Reply:
Do you mean the options of the two exe programs? I'm not sure how to write this section.

```
  virt-viewer.exe [OPTION?] -- ID|UUID|DOMAIN-NAME

Virtual machine graphical console

Help Options:
  -h, --help                                                           Show help options
  --help-all                                                           Show all help options
  --help-gtk                                                           Show GTK+ Options
  --help-gtk-vnc                                                       Show GTK-VNC Options
  --help-spice                                                         Show Spice Options

Application Options:
  -V, --version                                                        Display version information
  -z, --zoom=ZOOM                                                      Zoom level of window, in percentage
  -f, --full-screen                                                    Open in full screen mode (adjusts guest resolution to fit the client)
  -H, --hotkeys                                                        Customise hotkeys
  -r, --auto-resize=<never|always>                                     Automatically resize remote framebuffer
  -K, --keymap                                                         Remap keys format key=keymod+key e.g. F1=SHIFT+CTRL+F1,1=SHIFT+F1,ALT_L=Void
  --cursor=MODE                                                        Cursor display mode: 'local' or 'auto'
  -k, --kiosk                                                          Enable kiosk mode
  --kiosk-quit=<never|on-disconnect>                                   Quit on given condition in kiosk mode
  -v, --verbose                                                        Display verbose information
  --debug                                                              Display debugging information
  -d, --direct                                                         Direct connection with no automatic tunnels
  -a, --attach                                                         Attach to the local display using libvirt
  -c, --connect=URI                                                    Connect to hypervisor
  -w, --wait                                                           Wait for domain to start
  --reconnect                                                          Reconnect to domain upon restart
  --domain-name                                                        Select the virtual machine only by its name
  --id                                                                 Select the virtual machine only by its ID
  --uuid                                                               Select the virtual machine only by its UUID
  -s, --shared                                                         Share client session
```

```
  remote-viewer.exe [OPTION?] URI|VV-FILE

Remote viewer client
  -h, --help                                                           Show help options
Help Options:                                                          Show all help options
  -h, --help                                                           Show help options
  --help-all                                                           Show all help options
  --help-ovirt                                                         Show oVirt Options
  --help-gtk                                                           Show GTK+ Options
  --help-gtk-vnc                                                       Show GTK-VNC Options
  --help-spice                                                         Show Spice Optionsformation
  -z, --zoom=ZOOM                                                      Zoom level of window, in percentage
Application Options:                                                   Open in full screen mode (adjusts guest resolution to fit the client)
  -V, --version                                                        Display version information
  -z, --zoom=ZOOM                                                      Zoom level of window, in percentageffer
  -f, --full-screen                                                    Open in full screen mode (adjusts guest resolution to fit the client)_L=Void
  -H, --hotkeys                                                        Customise hotkeysde: 'local' or 'auto'
  -r, --auto-resize=<never|always>                                     Automatically resize remote framebuffer
  -K, --keymap                                                         Remap keys format key=keymod+key e.g. F1=SHIFT+CTRL+F1,1=SHIFT+F1,ALT_L=Void
  --cursor=MODE                                                        Cursor display mode: 'local' or 'auto'
  -k, --kiosk                                                          Enable kiosk mode information
  --kiosk-quit=<never|on-disconnect>                                   Quit on given condition in kiosk modeunnels
  -v, --verbose                                                        Display verbose information using libvirt
  --debug                                                              Display debugging information
  -t, --title                                                          Set window titleto start
  -s, --shared                                                         Share client sessionupon restart
  --domain-name                                                        Select the virtual machine only by its name
  --id                                                                 Select the virtual machine only by its ID
  --uuid                                                               Select the virtual machine only by its UUID
  -s, --shared                                                         Share client session
```